nanome/_internal/_plugin_instance_async.py

```diff
@@ -0,0 +1,27 @@
+from nanome.util import Logs
```

Comment:
Why is this method in its own file?

Reply:
To maintain Python 2 support; `async def` is a syntax error in py2

Reply:
I think you could use a 

> if sys.version_info >= (3, 4):
to prevent it from running on py2

Reply:
nah that wouldn't work because `async def` is invalid syntax in py2, checked at "compile" time